cmd/k3d/create.go

```diff
@@ -137,22 +134,22 @@ func runK3d(cmd *cobra.Command, args []string) error {
 	case "github":
 		key, err := internalssh.GetHostKey("github.com")
 		if err != nil {
-			return fmt.Errorf("known_hosts file does not exist - please run `ssh-keyscan github.com >> ~/.ssh/known_hosts` to remedy")
+			return errors.New("known_hosts file does not exist - please run `ssh-keyscan github.com >> ~/.ssh/known_hosts` to remedy")
```

Comment:
These are constant strings. While technically the win is minimal, a call to `errors.New()` is cheaper than a call to `fmt.Errorf()`